ui/raidboss/popup-text.ts

```diff
@@ -1184,7 +1184,9 @@ export class PopupText {
       let result = undefined;
       if (triggerHelper.triggerOptions.TTSText) {
         result = triggerHelper.valueOrFunction(triggerHelper.triggerOptions.TTSText);
-      } else if (triggerHelper.trigger.tts) {
+      } else if (triggerHelper.trigger.tts ||
+          triggerHelper.trigger.tts === null ||
+          triggerHelper.trigger.tts === '') {
```

Comment:
```suggestion
      } else if (triggerHelper.trigger.tts !== undefined) {
        // Allow null or false in this branch.
```

What about this to allow for null, false, and empty string?

Reply:
Would also allow `NaN` and `0`, if both of those are fine, then sure?

Reply:
I can go either way.  I think personally returning either of those here is sort of "undefined behavior" and it's fine if they do weird things.  If you think it's better to be explicit and ignore it, that seems fine too?

Reply:
Updated, with some more clear comments. Sorry for the double commit, I forgot to save the comments before I staged/commited